components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -406,22 +405,28 @@ SUBROUTINE p3_main_pre_main_loop(kts, kte, kbot, ktop, kdir, log_predictNc, dt,
           nc(k) = nccnst*inv_rho(k)
        endif
```

Comment:
I think the 4 lines above this are redundant with the activation code on line 423 below. Delete them?

Reply:
OK